src/pycmd.py

```diff
@@ -0,0 +1,95 @@
+"""Top level functions for turning Discord message input to Python command."""
+
+import io
+from contextlib import redirect_stdout
+
+
+fillerFile = 'filler.py'
```

Comment:
https://www.python.org/dev/peps/pep-0008/#constants

We will talk about contexts :)